README.md

```diff
@@ -1,5 +1,10 @@
 # MCDTool
 A tool to estimate ranks of alternatives and their uncertainties based on the Multi Criteria Decision Analysis approach.
+The variability of the MCDA scores are caused by:
```

Comment:
The variability come also from the weighting profiles!

Reply:
In the implementation now, I am expecting the user to give the weights, always. Otherwise it will receive an error message. I am not considering the possibility of assigning them randomly.

Reply:
However, in a next refinement, I'd like to add the possibility of assigning stochasticity to the weights too: each indicator will then have a distribution of weights assigned to it. I believe this to be a bit different than what is currently implemented in the R version, where a value is randomly sampled from a pdf if the user does not give any input value. Isn't it?